src/main/java/org/eolang/compiler/Program.java

```diff
@@ -67,8 +70,25 @@ public String java() throws IOException {
         );
         final TokenStream tokens = new CommonTokenStream(lexer);
         final ProgramParser parser = new ProgramParser(tokens);
-        parser.clauses();
-        return "done";
+        final Tree tree = parser.program().ret;
+        tree.java().entrySet().stream().forEach(
+            entry -> Program.save(
+                dir.resolve(entry.getKey()), entry.getValue()
+            )
+        );
+    }
+
+    /**
+     * Save content.
+     * @param file File to save to
+     * @param content Java content
+     */
+    private static void save(final Path file, final String content) {
```

Comment:
`saveToFile` or more descriptive name needed. Also complicates the class to have storage implemented in using class.